index.html

```diff
@@ -14,11 +14,15 @@ <h2>Contributing</h2>
 
 <h2>EIP status terms</h2>
 <ul>
-  <li><strong>Draft</strong> - an EIP that is open for consideration and is undergoing rapid iteration and changes.</li>
-  <li><strong>Last Call</strong> - an EIP that is done with its initial iteration and ready for review by a wide audience.</li>
-  <li><strong>Accepted</strong> - a core EIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author. The process for Core Devs to decide whether to encode an EIP into their clients as part of a hard fork is not part of the EIP process. If such a decision is made, the EIP will move to final. </li>
-  <li><strong>Final (non-Core)</strong> - an EIP that has been in Last Call for at least 2 weeks and any technical changes that were requested have been addressed by the author.</li>
-  <li><strong>Final (Core)</strong> - an EIP that the Core Devs have decided to implement and release in a future hard fork or has already been released in a hard fork.</li>
+  <li><strong>Idea</strong> - An idea that is pre-draft. This is not tracked within the EIP Repository.
+  <li><strong>Draft</strong> - The first formally tracked stage of an EIP in development. An EIP is merged by an EIP Editor into the EIP repository when properly formatted.</li>
+  <li><strong>Review</strong> - An EIP Author marks an EIP as ready for and requesting Peer Review.</li>
+  <li><strong>Last Call</strong> - This is the final review window for an EIP before moving to FINAL. An EIP editor will assign Last Call status and set a review end date (review-period-end), typically 14 days later. If this period results in necessary normative changes it will revert the EIP to REVIEW.</li>
+  <li><strong>Final</strong> - This EIP represents the final standard. A Final EIP exists in a state of finality and should only be updated to correct errata and add non-normative clarifications.</li>
+  <li><strong>Stagnant</strong> - Any EIP in DRAFT or REVIEW if inactive for a period of 6 months or greater is moved to STAGNANT. An EIP may be resurrected from this state by Authors or EIP Editors through moving it back to DRAFT.</li>
+  <li><strong>Withdrawn</strong> - The EIP Author(s) have withdrawn the proposed EIP. This state has finality and can no longer be resurrected using this EIP number. If the idea is pursued at later date it is considered a new proposal.</li>
+
+  <li><strong>Living</strong> - A special status for EIPs that are designed to be continually updated and not reach a state of finality. This includes most notably EIP-1. Any changes to these EIPs will move between REVIEW and LIVING states.</li>
```

Comment:
```suggestiotn
  <li><strong>Living</strong> - A special status for EIPs that are designed to be continually updated and not reach a state of finality. This includes most notably EIP-1.</li>
```
I believe we decided to remove the toggling of living.  We should fix EIP-1 if that is what it says.

Reply:
Yes - this is what EIP-1 says - I just copied from there to bring things en par

Reply:
Sorry, my comment was supposed to be a ```suggestion but I typod so it just ended up being a confusing comment.  Please have another look at the comment above.  I would rather have this be correct than copy EIP-1 (which is currently incorrect).  😊

Reply:
https://github.com/ethereum/EIPs/pull/3856